frontend/src/components/Pages/MissionPage/InpectionView/InspectionView.tsx

```diff
@@ -0,0 +1,332 @@
+
+const StyledInspection = styled.canvas`
+    flex: 1 0 0;
+    align-self: stretch;
+    max-height: 60vh;
+    width: 80vh;
+`
+
+const StyledInspectionImage = styled.canvas`
+    flex: 1 0 0;
+    align-self: center;
+    max-width: 100%;
+`
+
+const StyledDialog = styled(Dialog)`
+    display: flex;
+    width: 100%;
+    max-height: 80vh;
+`
+const StyledCloseButton = styled(Button)`
+    width: 24px;
+    height: 24px;
+`
+const StyledDialogContent = styled(Dialog.Content)`
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+`
+const StyledDialogHeader = styled.div`
+    display: flex;
+    padding: 16px;
+    justify-content: space-between;
+    align-items: center;
+    align-self: stretch;
+    border-bottom: 1px solid ${tokens.colors.ui.background__medium.hex};
+    height: 24px;
+`
+
+const StyledBottomContent = styled.div`
+    display: flex;
+    padding: 16px;
+    justify-content: space-between;
+    align-items: center;
+    align-self: stretch;
+`
+
+const StyledInfoContent = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+`
+
+const StyledSection = styled.div`
+    display: flex;
+    padding: 24px;
+    min-width: 240px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 8px;
+    border-radius: 6px;
+    border: 1.194px solid ${tokens.colors.ui.background__medium.hex};
+    background: ${tokens.colors.ui.background__default.hex};
+    overflow-y: scroll;
+    max-height: 60vh;
+`
+
+const StyledImagesSection = styled.div`
+    display: flex;
+    align-items: center;
+    gap: 16px;
+`
+
+const StyledImageCard = styled(Card)`
+    display: flex;
+    max-height: 280px;
+    max-width: 210px;
+    align-self: stretch;
+    padding: 4px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 2px;
+    flex: 1 0 0;
+    border-radius: 2px;
+    border: 1px solid ${tokens.colors.ui.background__medium.hex};
+    background: ${tokens.colors.ui.background__default.hex};
+    box-shadow:
+        0px 2.389px 4.778px 0px ${tokens.colors.ui.background__light.hex},
+        0px 3.583px 4.778px 0px ${tokens.colors.ui.background__light.hex};
+    cursor: pointer;
+`
+
+const StyledInspectionData = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    gap: 8px;
+`
+const StyledInspectionContent = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+`
+const StyledDialogInspectionView = styled.div`
+    display: flex;
+    flex-direction: row;
+    gap: 16px;
+`
+
+const StyledInspectionCards = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    align-content: flex-start;
+    gap: 8px;
+    align-self: stretch;
+    flex-wrap: wrap;
+`
+
+interface InspectionDialogViewProps {
+    task: Task
+    setInspectionTask: (inspectionTask: Task | undefined) => void
+    tasks: Task[]
```

Comment:
Would it be more descriptive with task -> selectedTask/displayedTask and tasks -> availableTasks or something? Its not very clear what the difference is